### PR TITLE
RunAsGroup KEP

### PR DESCRIPTION
**Pull request by krmayankk**

### CODE REVIEW

File: keps/sig-auth/20190131-runas-groupid.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+authors:
+  - "@krmayankk"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-node
+reviewers:
+  - "@smarterclayton"
+  - "@pmorie"
+  - "@php-coder"
+  - "@pweil-"
```
Comment (liggitt) [inline keps/sig-auth/20190131-runas-groupid.md]: this reviewer list is outdated, suggest @tallclair and @mrunalp

File: keps/sig-auth/20190131-runas-groupid.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+approvers:
+  - "@pmorie"
+  - "@smarterclayton"
+editor: TBD
+creation-date: 2017-06-21
+last-updated: 2017-09-14
+status: implemented
```
Comment (liggitt) [inline keps/sig-auth/20190131-runas-groupid.md]: `implementable`, still in beta

**Comment by liggitt** [comment]
/lgtm
/approve

/hold
for sig-node ack
/cc @mrunalp @derekwaynecarr

File: keps/sig-auth/20190131-runas-groupid.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+kep-number: 0
+title: RunAsGroup support in PodSpec and PodSecurityPolicy
+authors:
+  - "@krmayankk"
+owning-sig: sig-node
```
Comment (liggitt) [inline keps/sig-auth/20190131-runas-groupid.md]: nit: can you move the file to `keps/sig-node`?

**Comment by derekwaynecarr** [comment]
this is fine from my perspective.

/approve

File: keps/sig-node/20190131-runas-groupid.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+kep-number: 0
```
Comment (tallclair) [inline keps/sig-node/20190131-runas-groupid.md]: nit: delete this

File: keps/sig-node/20190131-runas-groupid.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  - sig-auth
+reviewers:
+  - "@tallclair"
+  - "@mrunalp"
+approvers:
+  - "@liggitt"
+  - "@derekwaynecarr"
+editor: TBD
+creation-date: 2017-06-21
+last-updated: 2017-09-14
```
Comment (tallclair) [inline keps/sig-node/20190131-runas-groupid.md]: please fix

File: keps/sig-node/20190131-runas-groupid.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+status: implementable
+---
+
+# RunAsGroup support in PodSpec and PodSecurityPolicy
+
+https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/runas-groupid.md
```
Comment (tallclair) [inline keps/sig-node/20190131-runas-groupid.md]: Can you copy most of the content from the design proposal to here? Preferably as a separate commit. The KEP should be the source-of-truth for the design.

File: keps/sig-node/20190131-runas-groupid.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+* [Graduation Criteria](#graduation-criteria)
+
+* [Implementation History](#implementation-history)
+
+
+## Graduation Criteria
+
+- Publish Test Results from Master Branch of Cri-o To http://prow.k8s.io [#72253](https://github.com/kubernetes/kubernetes/issues/72253)
+- Containerd and CRI-O tests included in k/k CI [#72287](https://github.com/kubernetes/kubernetes/issues/72287)
+- Make CRI tests failures as release informing
```
Comment (tallclair) [inline keps/sig-node/20190131-runas-groupid.md]: Are these the criteria for beta or GA?

Comment (tallclair) [inline keps/sig-node/20190131-runas-groupid.md]: Are there any validation changes that need to be made before going to beta?

Comment (liggitt) [reply keps/sig-node/20190131-runas-groupid.md]: no, validation was done properly in alpha to not require changes on promotion

**Comment by liggitt** [comment]
/hold cancel

Tim's comments would be good to address in a follow-up

**Comment by krmayankk** [comment]
@tallclair a follow up pr coming to address remaining comments

File: keps/sig-node/20190131-runas-groupid.md (hunk 7 above)
Comment (krmayankk) [reply keps/sig-node/20190131-runas-groupid.md]: Graduation i thought meant GA right ?

**Pull request merged**